package.json

```diff
@@ -90,7 +90,8 @@
     "coverageDirectory": "../coverage",
     "coveragePathIgnorePatterns": [
       "index.ts",
-      ".+\\/__tests__\\/.+\\.factory.(t|j)s"
+      ".+\\/__tests__\\/.+\\.factory.(t|j)s",
+      ".e2e.spec.ts"
```

Comment:
This should be `.e2e-spec.ts` right?

Reply:
Very true. It seems like Jest is forgiving and accepted the above. I've changed it to `.e2e-spec.ts` in 5101f9ec6788ebf8bd628877f0edaf8fe500d5f7, however.